app/stores/DocumentsStore.ts

```diff
@@ -281,6 +282,24 @@ export default class DocumentsStore extends BaseStore<Document> {
     });
   };
 
+  @action
+  fetchUsers = async (
```

Comment:
I don't think there is a _right_ answer of where to put this, but I would sway to towards the users store as the entity being requested and stored is a `User`.